src/client-services/github.client-service.ts

```diff
@@ -16,7 +14,6 @@ function axiosConfig(): AxiosInstance {
       'Content-Type': 'application/json',
     },
   });
```

Comment:
Was this auto formatting?

Reply:
No, i probably added it at some point. Is it ok?

Reply:
Leave it, that's fine. I just wanted to know.